mycroft/enclosure/gui.py

```diff
@@ -339,6 +340,37 @@ def show_url(self, url, override_idle=None,
         self.show_page("SYSTEM_UrlFrame.qml", override_idle,
                        override_animations)
 
+    def play_video(self, url, title="", override_idle=True):
+        """ Play video stream """
+        self["playStatus"] = "play"
+        self["video"] = url
+        self["title"] = title
+        self.video_info = {"title": title, "url": url}
+        self.show_page("SYSTEM_VideoPlayer.qml",
+                       override_idle=override_idle)
+
+    @property
+    def is_playing_video(self):
+        return self.video_info is not None
```

Comment:
Can the UI set the "playStatus" member to indicate that it has stopped playback? in that case should that also be considered?

Reply:
maybe add bus messages to clicks in the ui?

Reply:
this is essentially an internal method for use in pause and stop methods, and you are correct that is_video_displayed is more descriptive

that said i think it makes sense to expose this to users, and i think splitting into those two makes sense, will add those changes :+1:

Reply:
Yeah nice. 

With the addition of the `playback_status` would it be worth basing `is_video_displayed` off this status rather than the existence of `video_info`? 

`video_info` works, I'm just wondering if there's a higher chance of them getting out of sync with future changes. Such as adding a queuing method where `track_info` gets set to buffer a video prior to playback.